scrunch/session.py

```diff
@@ -0,0 +1,15 @@
+from pycrunch import connect as _connect
+from pycrunch.elements import ElementSession
+
+from .version import __version__
+
+
+class ScrunchSession(ElementSession):
+    headers = {
+        "user-agent": "scrunch/%s" % __version__
```

Comment:
We also want to see the pycrunch version, so append this to it rather than overwrite it.

Reply:
Added pycrunch's version as well